src/base/util/eventdispatcher.h

```diff
@@ -30,7 +30,7 @@ class EventDispatcher
 	typedef std::shared_ptr<Event> event_ptr;
 	typedef std::function<void(Params &)> handler_fn;
 	typedef std::map<std::string, event_ptr> event_map;
-	typedef std::list<std::pair<handler_id, handler_fn>> handler_list;
+	typedef std::list<std::pair<std::uint64_t, handler_fn>> handler_list;
```

Comment:
Do we still need handler_id then?

Reply:
This is part of the EventDispatcher refactor. Out of scope of this pr.